tests/app/main/test_services.py

```diff
@@ -2327,7 +2328,37 @@ def test_lists_correct_services_for_previous_framework_and_lot(self, get_metadat
 
         add_service_buttons = doc.xpath("//button[@class='button-secondary']")
         assert len(add_service_buttons) == 3
-        assert all(button.text == 'Add to G-Cloud 10' for button in add_service_buttons)
+        assert all(button.text == 'Add' for button in add_service_buttons)
+
+    @pytest.mark.parametrize(
+        ('lot_slug', 'lot_name'),
+        (
+            ('digital-outcomes', 'digital outcomes'),
+            ('digital-specialists', 'digital specialists'),
+            ('user-research-participants', 'user research participants'),
+        )
+    )
+    def test_shows_boolean_question_for_single_service_lots(self, get_metadata, data_api_client, lot_slug, lot_name):
+        data_api_client.get_framework.side_effect = [
+            self.framework(slug='digital-outcomes-and-specialists-3'),
+            self.framework(slug='digital-outcomes-and-specialists-2'),
+        ]
+        get_metadata.return_value = 'digital-outcomes-and-specialists-2'
+        data_api_client.find_services.return_value = {
+            'services': [{'serviceName': 'Service one', 'copiedToFollowingFramework': False}],
+        }
+
+        res = self.client.get(
+            f'/suppliers/frameworks/digital-outcomes-and-specialists-3/submissions/{lot_slug}/previous-services'
+        )
+        doc = html.fromstring(res.get_data(as_text=True))
+
+        assert res.status_code == 200
+        assert doc.xpath(f"//h1[normalize-space()='Do you want to reuse your previous {lot_name} service?']")
```

Comment:
Do we really have to have all our content duplicated into all our tests? No big deal, I know we do it everywhere. But I'd prefer to test "does the `h1` include `lot_name` or something like that.

Reply:
Yeah, I guess it might be a bit redundant. Just following what we do elsewhere.